src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/CompilationCreationTests.vb

```diff
@@ -2150,6 +2150,40 @@ End Class
             CheckCompilationSyntaxTrees(compilation4, tree2, tree3)
         End Sub
 
+        <Fact>
+        Public Sub CompilationDiagnosticsWorkForUndeclaredClassInLambdaFunction()
```

Comment:
>CompilationDiagnosticsWorkForUndeclaredClassInLambdaFunction [](http://example.com/codeflow?start=19&length=60)

Consider moving this test to src\Compilers\VisualBasic\Test\Emit\ExpressionTrees\CodeGenExprLambda.vb